tests/contrib/test_appengine.py

```diff
@@ -923,7 +921,7 @@ def test_decorator_from_client_secrets_bad_type(self):
         with loadfile_patch as loadfile_mock:
             loadfile_mock.return_value = ('badtype', None)
             self.assertRaises(
```

Comment:
> There are a lot more instances of this type of syntax in the file, so it may be another PR for that. 

please do.

> Should I go through and make similar changes for all of the test files? I'm thinking yes.

Don't sweat it too much. But if you're really bored, I won't stop you.

(One day I'll probably switch this project to py.test)